src/QMCTools/QMCGaussianParserBase.cpp

```diff
@@ -1566,6 +1575,7 @@ void QMCGaussianParserBase::createGridNode(int argc, char** argv)
             if(a == "-nojastrow")
```

Comment:
Remove the nojastrow option and *always* create both files .wfj and .wfnoj

I don't think space can be a concern due to the hdf5 options.

Reply:
Any Update on this? @ye-luo @prckent ? 
Since you are the only two who cared, what do you prefer? On file? or two files. If two files, which one goes in the Input.

Reply:
I prefer it simple and better if backward compatiable. Just one file. noj can be created manually by users in 2 secs.